applications/style.css

```diff
@@ -364,16 +367,55 @@ body {
 
 .application-details .application-details-actions .application-details-reject {
   color: var(--red-color);
-  height: auto;
   background-color: white;
-  margin-left: auto;
-  margin-right: auto;
-  text-decoration: underline;
+  margin: 0 auto;
+  width: 40%;
+  border: 2px solid var(--red-color);
+}
+
+.application-details
+  .application-details-actions
+  .application-details-reject:is(:hover, :active) {
+  background: var(--red-color);
+  color: var(--white);
+  transition: transform 0.2s ease;
+}
+
+.application-details
+  .application-details-actions
+  .application-details-reject:hover {
+  transform: scale(1.05);
```

Comment:
Including `transform` for the scaling effect is good, but keep in mind the performance implications of CSS animations on low-end devices. To optimize performance, you could use `will-change: transform`.

Reply:
Thanks for the feedback! I've added will-change: transform to optimize the scaling effect for better performance on low-end devices.